test/test_tree_2d_euler.jl

```diff
@@ -371,16 +371,16 @@ end
     @test_trixi_include(joinpath(EXAMPLES_DIR,
                                  "elixir_euler_sedov_blast_wave_sc_subcell.jl"),
                         l2=[
-                            0.4328635350273501,
-                            0.15011135840723572,
-                            0.15011135840723572,
-                            0.616129927549474,
+                            0.432804941135901,
```

Comment:
This is somewhat tougher to trace back to an origin.

But again, although the same number of timesteps is taken, the final timesteps are of different size:

1.9.4 

```julia
────────────────────────────────────────────────────────────────────────────────────────────────────
 Simulation running 'CompressibleEulerEquations2D' with DGSEM(polydeg=3)
────────────────────────────────────────────────────────────────────────────────────────────────────
 #timesteps:               1088                run time:       4.13231352e+01 s
 Δt:             1.23977582e-04                └── GC time:    5.14435900e-02 s (0.124%)
 sim. time:      2.00000000e+00 (100.000%)     time/DOF/rhs!:  1.37470074e-07 s
                                               PID:            1.82689698e-07 s
 #DOFs per field:         65536                alloc'd memory:        341.102 MiB
 #elements:                4096

 Variable:       rho              rho_v1           rho_v2           rho_e         
 L2 error:       1.07683817e+00   3.26884621e-01   3.26918606e-01   6.85853441e-01
 Linf error:     3.97702795e+00   1.76987846e+00   1.76997998e+00   2.92942191e+00
 ∑∂S/∂U ⋅ Uₜ :  -3.19044405e+01
────────────────────────────────────────────────────────────────────────────────────────────────────
```

1.10.2

```julia
────────────────────────────────────────────────────────────────────────────────────────────────────
 Simulation running 'CompressibleEulerEquations2D' with DGSEM(polydeg=3)
────────────────────────────────────────────────────────────────────────────────────────────────────
 #timesteps:               1088                run time:       3.93407320e+01 s
 Δt:             1.74772656e-04                └── GC time:    2.18495990e-02 s (0.056%)
 sim. time:      2.00000000e+00 (100.000%)     time/DOF/rhs!:  1.25852993e-07 s
                                               PID:            1.70317717e-07 s
 #DOFs per field:         65536                alloc'd memory:        392.484 MiB
 #elements:                4096

 Variable:       rho              rho_v1           rho_v2           rho_e         
 L2 error:       1.07686743e+00   3.26907413e-01   3.26889036e-01   6.85857082e-01
 Linf error:     4.00149876e+00   1.78189710e+00   1.78167204e+00   2.92940374e+00
 ∑∂S/∂U ⋅ Uₜ :  -3.18236709e+01
────────────────────────────────────────────────────────────────────────────────────────────────────
```

Reply:
Maybe also try this one with a non-adaptive scheme?

Reply:
For some reason this does not work when not using `stepsize_callback` but just using a small timestep.

Maybe @bennibolm knows why?

Reply:
Probably due to the specialized time stepping scheme. But then, if it is already using the stepsize_callback, can you just chose a standard RK scheme instead of an adaptive one?

Reply:
This example is using the custom Trixi SSPRK33 for application of stage limiters. Thus, when using a different integrator this feature is lost and the testcase is quite a different one (not sure if it even runs)

Reply:
Ah, right.

@bennibolm @amrueda Would any of you like to take a stab at this and investigate the source of the differences or are you ok with updating the reference values? Our current best guess is that everything seems ok and that the differences are mostly due to accumulated FP differences.

Reply:
> For some reason this does not work when not using `stepsize_callback` but just using a small timestep.

That is actually weird. I will take a look at it and hopefully fix it. It seems like it doesn't matter which `dt` is set manually.

I often had problems with the tests for both blast elixirs. I never really found a reason for that, just accepted and changed the numbers when necessary. So, I would have no problem with just adapting the expecting errors.

Reply:
OK, thanks for your feedback. Then, maybe we can proceed as follows:
* @DanielDoehring you can go ahead by just updating the reference values for the affected `sc_subcell` elixirs
* @bennibolm you will create an issue for the bug you just mentioned to keep track of it. Once you get to it, you might update the reference values here once more

Does that sound ok for you, @bennibolm

Reply:
See https://github.com/trixi-framework/Trixi.jl/pull/1919

Reply:
OK, thanks!